dotCMS/src/main/java/com/dotcms/vanityurl/business/VanityUrlAPIImpl.java

```diff
@@ -388,5 +388,16 @@ public boolean handleVanityURLRedirects(final VanityUrlRequestWrapper request, f
       return false;
   }
 
+    @Override
+    @CloseDBIfOpened
+    public List<CachedVanityUrl> findByForward(final Host host, final Language language, final String forward,
+                                               int action) {
+        return load(host, language)
+                .stream()
+                .filter(cachedVanityUrl -> cachedVanityUrl.response == action)
```

Comment:
I might be wrong, but shouldn't `.equals()` be used instead of `==` operator?

Reply:
Not because they are ints